tests/queries/0_stateless/03458_wkb_function.sql

```diff
@@ -0,0 +1,31 @@
+SELECT readWKBPoint(unhex('010100000000000000000000000000000000000000'));
```

Comment:
Please add tests with invalid inputs and check that the errors returned are appropiate.

Reply:
I thought that stateless tests can't check errors messages, isn't it?

Reply:
You can by adding a comment with the expected error, for example:

https://github.com/ClickHouse/ClickHouse/blob/c61eef4a7659e4856cc3266d8d7dd28b4e095d2b/tests/queries/0_stateless/03034_dynamic_conversions.sql#L10